src/components/pages/pages.stories.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import { Pages, Page } from './pages';
+import DialogFullScreen from '../dialog-full-screen/dialog-full-screen';
+import Heading from '../heading/heading';
+import Button from '../button/button';
+
+const store = new Store({
+  open: false,
+  slideIndex: 0
+});
+
+const handleSlide = (ev) => {
+  action('slide')();
+  store.set({ slideIndex: (ev.target.name || 0) });
+};
+
+const handleOpen = () => {
+  action('open')();
+  store.set({ open: true });
+};
+
+const handleCancel = () => {
+  action('cancel')();
+  store.set({ open: false });
+};
+
+const CustomState = (props) => {
+  return (
+    <State store={ store }>
+      { props.children }
+    </State>
+  );
+};
+
+CustomState.propTypes = {
+  children: PropTypes.node
+};
+
+const DialogState = props => new CustomState(props);
+const PageState = props => new CustomState(props);
+
+
+storiesOf('Pages', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [Button, DialogFullScreen, State]
+    }
+  })
+  .add('default', () => {
+    return (
```

Comment:
I don't think we should see this.

![image](https://user-images.githubusercontent.com/26925206/53879642-e8ba0f80-4006-11e9-9b9e-288221c40c26.png)

Reply:
Removed 👍